src/components/SqlEditor.tsx

```diff
@@ -105,26 +124,33 @@ export const SQLEditor = (props: SQLEditorProps) => {
   };
 
   return (
-    <div className={styles.Common.wrapper}>
-      <a
-        onClick={() => onToggleExpand()}
-        className={styles.Common.expand}
-        data-testid={selectors.components.QueryEditor.CodeEditor.Expand}
-      >
-        <i className={`fa fa-${expand.icon}`}></i>
-      </a>
-      <CodeEditor
-        aria-label="SQL"
-        height={expand.height}
-        language="sql"
-        value={query.rawSql || ''}
-        onSave={onSqlChange}
-        showMiniMap={false}
-        showLineNumbers={true}
-        onBlur={(text) => onChange({ ...query, rawSql: text })}
-        onEditorDidMount={(editor: any) => handleMount(editor)}
-      />
-    </div>
+    <>
+      { app === CoreApp.Explore &&
```

Comment:
In explore view we show the query type so that the user can pick which format/panel will appear. This isn't necessary in dashboard view since they manually select their preferred panel

![image](https://github.com/grafana/clickhouse-datasource/assets/28887171/366ad2f2-6488-444c-be7d-8fc595e6d85d)

Reply:
It's a good candidate for an additional comment when someone asks a `Why?` question. Please, consider adding a comment